mmv1/templates/terraform/examples/privileged_access_manager_entitlement_basic.tf.erb

```diff
@@ -0,0 +1,39 @@
+resource "google_privileged_access_manager_entitlement" "<%= ctx[:primary_resource_id] %>" {
+    provider =  google-beta
+    entitlement_id = "<%= ctx[:vars]['entitlement_id'] %>"
+    location = "global"
+    max_request_duration = "43200s"
+    parent = "projects/ci-test-project-188019"
+    requester_justification_config {    
+        unstructured{}
+    }
+    eligible_users {
+        principals = ["serviceAccount:test2102@itsvarsharma-pam-testing.iam.gserviceaccount.com"]
```

Comment:
This may not work. Change this to a user ldap (mainly, this domain itsvarsharma-pam-testing.iam.gserviceaccount.com will be unknown to the test project), we should use an SA from this org, or use a gmail / google user account.